Week-1/assignment-2/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+        href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+    <link rel="stylesheet" href="./scss/style.css">
+    <title>Audio Player</title>
+</head>
+
+<body>
+    <div class="parent-container">
```

Comment:
@chinmay4o instead of using class, please use web semantics with appropriate tags(header, nav, sections,etc)

Reply:
added sections